openapi.yaml

```diff
@@ -754,12 +815,16 @@ components:
           type: string
         public:
           type: boolean
+          default: true
```

Comment:
is this actually implemented? 
also this block seems to be duplicated here https://github.com/mo-pm888/twitter/pull/48/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R473